2024/wrestrc_waypoints_2024.csv

```diff
@@ -0,0 +1,13 @@
+Alpha,43.4338974,-80.5773193
+Bravo,43.433985,-80.577853
+Charlie,43.4341087,-80.5783774
+Delta,43.434172,-80.5786456
+Echo,43.4341389,-80.5788347
+Foxtrot,43.4340152,-80.5789286
+Golf,43.4336072,-80.5790949
+Hotel,43.4334913,-80.579001
+India,43.4333101,-80.5783559
+Juliette,43.4333549,-80.5782044
+Kilo,43.4337834,-80.5779791
+Lima,43.4338214,-80.5778061
+Mike,43.4337367,-80.5773863
```

Comment:
Change name column from NATO alphabet to regular increasing numbers (1, 2, 3, etc.).